internal/repos/importer.go

```diff
@@ -127,12 +134,14 @@ func (i *importer) handleGitHubImport(ctx context.Context, imp db.ListRepoImport
 		return err
 	}
 
-	// start a new pg transaction to handle the repo upsert
+	// start a new pg transaction to handle the repo upsert  and handle deletion of removed repos
```

Comment:
```suggestion
	// start a new pg transaction to handle the repo upsert and handle deletion of removed repos
```

just removing an extra space